virttest/iscsi.py

```diff
@@ -824,6 +826,17 @@ class Iscsi(object):
     def create_iSCSI(params, root_dir=data_dir.get_tmp_dir()):
         iscsi_instance = None
         err_msg = "Please install package(s): %s"
+        ubuntu = distro.detect().name == 'Ubuntu'
```

Comment:
try to clear the configuration before using iscsi if need(not sure)

Reply:
@prudmirya Thanks! agreed it is good to consider clearing the configuration before using iscsi, but please fix it from PR #1042, This patch will handle the pre-requisite of having the packages installed to use iscsi.